### DISCUSSION TITLE
Choose a specific font weight or font style in NextJs 13

### DISCUSSION THREAD

**Discussion by ChinonsoIg** [vercel/next.js#45983] [Help]
I am using Next 13 google fonts.
From the docs, you can specify an array of values for the font weight and font style like so:
```
import { Montserrat } from '@next/font/google'

const inter = Montserrat({ 
  weight: ['300', '900'],
  style: ['normal', 'italic'],
  subsets: ['latin'] 
})
```
But, it did not specify how one can reference a particular font weight or font style. I have tried out a couple of things , but no solution.

Anyone know how to go about this?

**Comment by dluan** [comment]
> ```
> import { Montserrat } from '@next/font/google'
> 
> const inter = Montserrat({ 
>   weight: ['300', '900'],
>   style: ['normal', 'italic'],
>   subsets: ['latin'] 
> })
> ```


This method works for preloading the fonts, so you just need to define what weights and styles you need. In the CSS you define what you need.

See if this example works for you. https://nextjs.org/docs/api-reference/next/font#css-variables
